package.json

```diff
@@ -48,7 +48,7 @@
   "main": "webpack.config.js",
   "keywords": [],
   "dependencies": {
-    "identity-style-guide": "^0.2.4",
+    "identity-style-guide": "18f/identity-style-guide#bh-add-app-styles",
```

Comment:
Also, I'm switching the version to be an exact version number instead of always grabbing the latest just so we have a bit more control over things while we're doing these big shifts.